deploy-manage/deploy/cloud-on-k8s/install-using-yaml-manifest-quickstart.md

```diff
@@ -14,5 +17,79 @@ mapped_urls:
+
+Before you begin, review the following prerequisites and recommendations:
+
+* For this quickstart guide, your Kubernetes cluster is assumed to be already up and running. Before you proceed with the ECK installation, make sure you check the [supported versions](/deploy-manage/deploy/cloud-on-k8s.md#k8s-supported).
+
+* If you are using GKE, make sure your user has `cluster-admin` permissions. For more information, check [Prerequisites for using Kubernetes RBAC on GKE](https://cloud.google.com/kubernetes-engine/docs/how-to/role-based-access-control#iam-rolebinding-bootstrap).
+
+* If you are using Amazon EKS, make sure the Kubernetes control plane is allowed to communicate with the Kubernetes nodes on port 443. This is required for communication with the Validating Webhook. For more information, check [Recommended inbound traffic](https://docs.aws.amazon.com/eks/latest/userguide/sec-group-reqs.md).
+
+* Refer to [*Install ECK*](../../../deploy-manage/deploy/cloud-on-k8s/install.md) for more information on installation options.
+
+* Check the [upgrade notes](../../../deploy-manage/upgrade/orchestrator/upgrade-cloud-on-k8s.md) if you are attempting to upgrade an existing ECK deployment.
+
+##  Installation procedure
+
+To deploy the ECK operator:
+
+1. Install [custom resource definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) with [`create`](https://kubernetes.io/docs/reference/kubectl/generated/kubectl_create/):
+
+    ```sh
+    kubectl create -f https://download.elastic.co/downloads/eck/{{eck_version}}/crds.yaml
+    ```
+
+    This will output similar to the following upon Elastic resources' creation:
+
+    ```sh
+    customresourcedefinition.apiextensions.k8s.io/agents.agent.k8s.elastic.co created
+    customresourcedefinition.apiextensions.k8s.io/apmservers.apm.k8s.elastic.co created
+    customresourcedefinition.apiextensions.k8s.io/beats.beat.k8s.elastic.co created
+    customresourcedefinition.apiextensions.k8s.io/elasticmapsservers.maps.k8s.elastic.co created
+    customresourcedefinition.apiextensions.k8s.io/elasticsearches.elasticsearch.k8s.elastic.co created
+    customresourcedefinition.apiextensions.k8s.io/enterprisesearches.enterprisesearch.k8s.elastic.co created
+    customresourcedefinition.apiextensions.k8s.io/kibanas.kibana.k8s.elastic.co created
+    customresourcedefinition.apiextensions.k8s.io/logstashes.logstash.k8s.elastic.co created
+    ```
+
+2. Install the operator with its RBAC rules with [`apply`](https://kubernetes.io/docs/reference/kubectl/generated/kubectl_apply/):
+
+    ```sh
+    kubectl apply -f https://download.elastic.co/downloads/eck/{{eck_version}}/operator.yaml
+    ```
+
+    ::::{note}
+    The ECK operator runs by default in the `elastic-system` namespace. It is recommended that you choose a dedicated namespace for your workloads (such as Elasticsearch and Kibana), rather than using the `elastic-system` or the `default` namespace.
```

Comment:
the brackets don't make sense to me. do you mean the person should create an `elasticsearch` and `kibana` namespace? a dedicated namespace for elasticsearch and kibana? how is "workloads" defined?

Reply:
workloads = applications (in this context orchestrated by the operator) = Elastic Stack apps = Elasticsearch and Kibana (among others of course).

I added the brackets for people that might not be familiar with the kubernetes term of `workloads`. But I'll remove it and leave the original text. The entire note doesn't add much value actually.

Maybe we should just mention to not orchestrate applications in `elastic-system` namespace instead.

Reply:
I've updated it to:

```
    ::::{note}
    The ECK operator runs by default in the `elastic-system` namespace. While this namespace is used for the operator itself, it is recommended that you deploy your application workloads in a separate, dedicated namespace instead of `elastic-system` or `default`. You will need to consider this when setting up your applications.
    ::::
```

I think it's much clearer now :)